packages/imtr-client/src/types.ts

```diff
@@ -3,7 +3,9 @@ import Question from "./models/question";
 
 export type Input = Question | Decision;
 export type ClientSimpleType = boolean | string | number;
-export type Answer = ClientSimpleType | undefined;
+export type AnswerType = Answer | undefined;
+export type Answer = ClientSimpleType | string[];
```

Comment:
Why name this `AnswerType`? I think that's confusing, because I don't understand why this is a `type`. Why not just:
`export type Answer = ClientSimpleType | string[] | undefined;` 
or rename to something that's more understandable

Reply:
An Answer is available if a question is actually answered. The AnswerType includes the option un-answerd.

Reply:
It's to distinct between answered (Answer) and potentially un-answered (AnswerType) questions. Any suggestions for  better names?

Reply:
`AnswerType` (or the original `Answer`) could be renamed to `AnsweredAnswer`, `UserAnswer`, `GivenAnswer`, `QuestionAnswer`?